include/dlaf/eigensolver/reduction_to_band/impl.h

```diff
@@ -209,40 +209,36 @@ void updateTrailingPanel(const bool has_head, const std::vector<TileT<T>>& panel
   }
 }
 
-template <Backend B, typename ASender, typename WSender, typename XSender>
+template <Backend B, typename T, typename ASender, typename WSender, typename XSender>
 void hemmDiag(pika::threads::thread_priority priority, ASender&& tile_a, WSender&& tile_w,
               XSender&& tile_x) {
-  using T = dlaf::internal::SenderElementType<ASender>;
```

Comment:
When senders were first introduced we discussed about it and chose not to add an extra template parameter.
What did change?